iocage_lib/ioc_start.py

```diff
@@ -1312,19 +1311,27 @@ def __check_rtsold__(self):
             stdout=su.PIPE
         )
 
-    def get_default_gateway(self):
-        # e.g response - ('192.168.122.1', 'lagg0')
-        try:
-            return netifaces.gateways()["default"][netifaces.AF_INET]
-        except KeyError:
+    def get_default_interface(self):
+        if netifaces.AF_INET in self.host_gateways['default']:
+            return self.host_gateways['default'][netifaces.AF_INET][1]
+        elif netifaces.AF_INET6 in self.host_gateways['default']:
+            return self.host_gateways['default'][netifaces.AF_INET6][1]
+        else:
             iocage_lib.ioc_common.logit(
                 {
                     'level': 'EXCEPTION',
-                    'message': 'No default gateway interface found'
+                    'message': 'No default interface found'
                 },
                 _callback=self.callback,
-                silent=self.silent
-            )
+                silent=self.silent)
+
+    def get_default_gateway(self, address_family=netifaces.AF_INET):
+        if address_family in self.host_gateways['default']:
+            return re.sub(
+                '%.*$', '%epair0b',
+                self.host_gateways['default'][address_family][0])
+        else:
+            return ''
```

Comment:
This should return `none` i think which will indicate nothing can be found for this and hence we don't perform anything for `none` in our starting setup

Reply:
This would also make sure that we don't set empty string on start up